### ISSUE TITLE
Reguarding: "Mobile SOS"

### ISSUE THREAD

**Issue by rozzzly** [jaggy/ideas#1]
I've got this idea from watching 100s of videos about police brutality; it's disgusting the abuses of power that go on everyday. Now, 98% of cops are great people doing their jobs, trying to keep their community safe. But what about the other 2%? monsters. And unless you have real video proof (and this often doesn't mean shit anyway) you've got no recourse to seek justice. 

So here's my idea, which overlaps with your "Mobile SOS" idea, @jaggy. 

You have a really quick way to put your phone into a mode where it does 2 things:
1. **Start to record video/sound from both cameras of your smart phone WITHOUT the screen being on.** (that is possible with the android api, I would imagine on iOS too)
   - If you are holding the phone up in front of a cops face it'll most likely make them more angry which could result in damage to your person
   - they might try to destroy or confiscate your phone so you can't use that against them. **This happens all the fucking time.  Go look it up on YouTube.** 
     - through a process called civil forfeiture, a cop can confiscate you're property claiming its relevant to an ongoing investigation. If that happens, it's very very unlikely you'll get your property back. All the cop has to say it, "I think I smelled weed in the car and this guy looks like he might be trafficking marijuana and heroin so we're going to confiscate his phone because hes probably using it to sell drugs" Seriously this happens, and the reason given is often not much more legit that that. It doesn't have to have any bias in truth, because they're only "investigating"
2. **Lock the phone.** Most of us atleast use a pin / pattern for authentication, but if you've used the phone in the last, say 15 minutes, it could still very well be unlocked allowing police access to all of your files/texts/etc. The 4th amendment requires that police have a warrant to search your device, but if they handcuff you, and pull out your unlocked phone.... well you can't really stop them. They could delete the video of them beating you and you would have no proof. If you wanted to seek justice, you would need a lawyer, spend 10s of thousands of dollars, and hundreds of hours to even have the chance of fighting police abuse in court. 

---

> #### Consider this situation:
> 
> If you're a cop and you hear "Officer Bob beat me for no reason." Then Officer Bob says, "the _criminal_ was acting like they had a weapon, and when I tried to intervene they resisted arrest, so I had no choice but to subdue them."  What side do you believe? Your co-worker who you know and trust, or this random person who apparently is a criminal?  You'll most likely side with Officer Bob. Moreover, even if you are suspicious that there might have been some user of excessive force, do you want to be the one to stand up in-front of you're department and call the other officer into question? probably not, because the entire department will retaliate against you. 
> 
> I've often heard this referred to as the _"Code of Silence", "Blue Code", etc._ This is a real issue, and happens way more than it should. But without proof, the average citizen doesn't really have anything they can do about it unless they can mount a multimillion dollar lawsuit. **That is, unless you have video of the abuse occurring which you can show to the public and demand justice.**

**Comment by rozzzly** [comment]
It'd be really cool if there was some non-profit like the ACLU (hell google might even make the servers available) would offer a public service so that when the video starts recording, it gets streamed to some server so destroying the device, or deleting the file cant prevent the truth from getting out. 

this has a bunch of applications beyond just police; this could be super effective in domestic abuse cases.

Anyway, It would just be awesome to have a google now command, "Cop Block" or some button combo you can trigger to get the camera rolling because opening the phone, scrolling through an app, launching it, etc, etc, etc takes way to long and could prevent you from recording potential abuse.

**Comment by jaggy** [comment]
Yeah, I understand what you mean and safety is such a real issue nowadays.

Regarding the phone screen being off while recording, I'm not really sure IOS can support that on that end without being jailbroken though.

I remember seeing [Matt Stauffer](https://twitter.com/stauffermatt) tweeting of a prototype where you dial a phone number hosted from Twilio and it records the call.

After placing the call, if the caller doesn't click the link sent to them, it'll immediately be sent out to people you've registered with your account.

The idea was, when you're pulled over by a cop, you dial the number and it records the interaction, then it can potentially be send to people as evidence.

I'll look for the link since I might be butchering what it does.

**Comment by jaggy** [comment]
> A simple voice recording via a phone call gives the user an opportunity to log essential details of where they are and what led to the stop. Once the call is terminated and the user’s friends and family are notified, they can act quickly based on the information.

@rozzzly, the app is called [PulledOver](https://www.twilio.com/blog/2016/03/the-app-that-shouldnt-exist-pulled-over.html).

**Comment by jaggy** [comment]
@rozzzly Yeah, streaming is one of the best ways for reactive responses rather than hearing/seeing about it only after it happened.
